Integration/FRAIHMWORK Integrator Starter Kit Collection.json

```diff
@@ -0,0 +1,2038 @@
+												"exec": [
+													""
+												],
+												"type": "text/javascript"
+											}
+										}
+									],
+									"request": {
+										"auth": {
+											"type": "bearer",
+											"bearer": [
+												{
+													"key": "token",
+													"value": "{{access_token}}",
+													"type": "string"
+												}
+											]
+										},
+										"method": "GET",
+										"header": [],
+										"url": {
+											"raw": "{{baseUrl}}{{Monitoring}}/fault/{{UUIDFaultResponse}}",
+											"host": [
+												"{{baseUrl}}{{Monitoring}}"
+											],
+											"path": [
+												"fault",
+												"{{UUIDFaultResponse}}"
+											]
+										}
+									},
+									"response": []
+								}
+							]
+						},
+						{
+							"name": "PUT",
+							"item": [
+								{
+									"name": "Updates a fault that is affecting a component",
+									"event": [
+										{
+											"listen": "prerequest",
+											"script": {
+												"exec": [
+													""
+												],
+												"type": "text/javascript"
+											}
+										},
+										{
+											"listen": "test",
+											"script": {
+												"exec": [
+													""
+												],
+												"type": "text/javascript"
+											}
+										}
+									],
+									"request": {
+										"auth": {
+											"type": "bearer",
+											"bearer": [
+												{
+													"key": "token",
+													"value": "{{access_token}}",
+													"type": "string"
+												}
+											]
+										},
+										"method": "PUT",
+										"header": [
+											{
+												"key": "Content-Type",
+												"value": "application/json"
+											}
+										],
+										"body": {
+											"mode": "raw",
+											"raw": "{\n    \"name\": \"Fault of a component\",\n    \"code\": 0,\n    \"severity\": \"MEDIUM\",\n    \"uuid\": \"{{UUIDFaultResponse}}\",\n    \"description\": \"Description of a fault\",\n    \"source\": \"{{UUIDComponentResponse}}\",\n    \"timeOfDetection\": \"2020-08-05T21:57:44.823Z\",\n    \"timeOfValidity\": \"2020-08-05T21:57:44.823Z\",\n    \"timeout\": 100,\n    \"extensions\": {\n        \"property\" :        {\n            \"name\": \"{{String}}\",\n            \"value\": \"{{String}}\",\n            \"type\": \"INTEGER\"\n        }\n    }\n}"
+										},
+										"url": {
+											"raw": "{{baseUrl}}{{Monitoring}}/fault/:uuid",
+											"host": [
+												"{{baseUrl}}{{Monitoring}}"
+											],
+											"path": [
+												"fault",
+												":uuid"
+											],
+											"variable": [
+												{
+													"key": "uuid",
+													"value": "{{UUIDFaultResponse}}",
+													"description": "(Required) Uuid of the fault being updated"
+												}
+											]
+										},
+										"description": "Allows a component to report one or more faults currently affecting the component"
```

Comment:
I think the documentation here is referencing the original API spec, and needs updating